data/QChem/basicQChem4.2/dvb_bomd.in

```diff
@@ -0,0 +1,37 @@
+$comment
```

Comment:
Do we already have the input/output for some program?

Reply:
I copied the Gaussian BOMD output from regressions. That's it.

Reply:
OK, that's fine. Do we have one for Gaussian as well? I believe it has support for BO MD. 

Also, will we be deleting the regression files after this gets in?

Reply:
Yes, there's a PR. Working on recreating the appropriate Gaussian input, which isn't present.